### ISSUE TITLE
Document Tree Search function not working 

### ISSUE THREAD

**Issue by RotoruaNZ2023** [KenticoDevTrev/DynamicRouting#59]
Returns this error message. Error occurred: 'Internal Server Error'. Please check the event log. Do you want to refresh the UI?

**Comment by RotoruaNZ2023** [comment]
Looking at the stack trace, the error is coming from "DynamicRouting.Kentico.Base.DynamicRoutingDocumentURLProvider.GetPresentationUrlInternal"

**Comment by KenticoDevTrev** [comment]
Do you have the presentation URL set in the site application?

**Comment by RotoruaNZ2023** [comment]
Where do I check that? I am not a developer sorry.

**Comment by KenticoDevTrev** [comment]
Log into the admin of kentico and go to sites, see if you have the proper site presentation URL.

I will say that if you don't know that much about kentico, you should look to either take their online course or get aid from a kentico partner.

**Comment by RotoruaNZ2023** [comment]
Yes, we have a Kentico partner and I will share this thread with them. Thank you.
